### DISCUSSION TITLE
说句闲话

### DISCUSSION THREAD

**Discussion by langningchen** [langningchen/XMOJ-Script#134] [General]
@boomzero @PythonSmall-Q 
最近的Git Graph真壮观啊！

![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/902b1235-7fa1-49f3-a069-d95d80a4345a)
![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/08262bc6-7009-4919-9947-76610aee7a45)
![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/e5659a07-fdd7-4ff2-9054-1802439cd0e8)

**Comment by boomzero** [comment]
az, 6

**Comment by boomzero** [comment]
![](https://github.com/langningchen/XMOJ-Script/assets/85378277/9c30dc52-8d32-493c-857c-1cd3a36ff75e)
? 没有啊

**Comment by langningchen** [comment]
有一些我本地的

**Comment by langningchen** [comment]
另外，你是选择的All branches吗？

**Reply by boomzero** [reply]
PHPStorm... 默认的吧

**Comment by PythonSmall-Q** [comment]
6

**Comment by PythonSmall-Q** [comment]
相信这个项目会越来越好

**Comment by PythonSmall-Q** [comment]
说句闲话，我VPN过期了艹

**Reply by PythonSmall-Q** [reply]
@langningchen

**Reply by langningchen** [reply]
那怎么办？

**Comment by boomzero** [comment]
说句闲话
![mail](https://github.com/langningchen/XMOJ-Script/assets/85378277/89d766a0-8597-420c-8eb7-fbf99612515f)
发生了什么, 我的mail都卡死了...

**Reply by langningchen** [reply]
至少比我那里好，因为我是管理员，所以一般我一天能收到1000多封邮件

**Reply by boomzero** [reply]
@langningchen 我的adlist仓库的actions每fail一次github就发邮件, 所以我也有几千封邮件

**Reply by boomzero** [reply]
只不过我写了脚本自动删

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 再说句闲话：你们今天没有返校吗？

**Reply by boomzero** [reply]
2 days ago 没有

**Reply by langningchen** [reply]
好吧 @boomzero

**Comment by boomzero** [comment]
@langningchen 说句闲话, 为什么我 1,598,096 ++    1,595,125 --

**Reply by boomzero** [reply]
commits

**Reply by langningchen** [reply]
@boomzero 那可能又是格式化了

**Reply by boomzero** [reply]
...

**Reply by boomzero** [reply]
@langningchen 那个commit 进master了...

**Reply by langningchen** [reply]
哪个 @boomzero 就是格式化的那个吗？

**Reply by boomzero** [reply]
I have no idea, 但是你看code frequency @langningchen

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 有一个很严重的问题

**Comment by langningchen** [comment]
![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/b111ae2c-8f23-424b-a295-4e0e9b31596f)

**Comment by langningchen** [comment]
![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/efb2f032-843d-47d6-a9a2-ab514791d173)

**Comment by langningchen** [comment]
不过我现在不用xmldom了，应该还好吧？ @boomzero @PythonSmall-Q

**Reply by boomzero** [reply]
~upgrade?~

**Reply by langningchen** [reply]
@boomzero 我用了另一个库

**Reply by boomzero** [reply]
删掉依赖?

---
Delete dependencies?

**Reply by langningchen** [reply]
我的package.json已经改了。但是package-lock.json没有改

---
My package.json has been changed. But package-lock.json has not changed

**Comment by boomzero** [comment]
@langningchen ~为什么我们的网站在美国飞快, 在中国非常慢...~
![](https://github.com/langningchen/XMOJ-Script/assets/85378277/032c9ac8-5d18-404e-a74f-d9e12386aaa7)

**Reply by langningchen** [reply]
emm

**Comment by PythonSmall-Q** [comment]
因为cloudflare服务器在美国。Cdn不会加速很多的

Sent from Mail for Windows

**Comment by boomzero** [comment]
再说句闲话, ping真智能
![ping](https://github.com/langningchen/XMOJ-Script/assets/85378277/cc285908-85d5-4df6-9e71-8992b6b9c090)

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 你知道吗？我们学校周一的升旗仪式演讲的同学说：“如果你用代理服务器访问境外网站，你可能会受到DDOS攻击”

**Reply by boomzero** [reply]
~什么主题的升旗仪式,说这个~

**Reply by boomzero** [reply]
应该是~你的服务器可能受到gfw的ddos~

**Reply by langningchen** [reply]
> ~什么主题的升旗仪式,说这个~

网络安全靠人民，网络安全为人民 @boomzero

**Reply by langningchen** [reply]
> 应该是~你的服务器可能受到gfw的ddos~

Emm

**Comment by langningchen** [comment]
他知道的很多，但是不够多

**Comment by langningchen** [comment]
他不知道DDOS无法攻击个人电脑，只能攻击服务器

**Comment by boomzero** [comment]
我们升旗仪式老师提到我教她用chatgpt...

**Comment by boomzero** [comment]
CCF的服务器真稳定啊, 一个准考证都下载不了

**Reply by boomzero** [reply]
trick:
在console 执行
```JS
downloadAdmission_stu();
```

**Reply by langningchen** [reply]
NOI的网站太好了！全是500！@boomzero 

![20230914_070732.jpg](https://github.com/langningchen/XMOJ-Script/assets/81672884/210b6c1a-59e9-472c-b684-f2eb3b100593)

**Reply by langningchen** [reply]
我可是好不容易才登录进去的

**Reply by boomzero** [reply]
@langningchen 

我们的网站..
![](https://github.com/langningchen/XMOJ-Script/assets/85378277/3b4c7969-577f-4ffd-8fe4-668a1fde891b)

**Reply by langningchen** [reply]
@boomzero 为什么`5XX`？

**Reply by boomzero** [reply]
@langningchen fixed by #407

**Reply by boomzero** [reply]
因为他在爬后台

**Reply by langningchen** [reply]
@boomzero 我们可以弄一个sitemap

**Reply by PythonSmall-Q** [reply]
sitemap应该会比较简单（的吧

**Reply by langningchen** [reply]
@PythonSmall-Q 你考下午的吗？

**Reply by langningchen** [reply]
我要走了

**Comment by PythonSmall-Q** [comment]
考。

Sent from Mail for Windows

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 我发现 Snipping Tool 可以识别文字了！而且还可以录屏

![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/a06ec4d2-6e03-41b9-868a-082347a24dde)

注意我的电脑版本

![image](https://github.com/langningchen/XMOJ-Script/assets/81672884/2510d1aa-632d-4547-9af0-de19af5f2761)

**Comment by boomzero** [comment]
preview也可以

**Comment by boomzero** [comment]
![image](https://github.com/langningchen/XMOJ-Script/assets/85378277/3c30843a-179e-49ee-8e3a-1cab9fed47de)

**Reply by boomzero** [reply]
![image](https://github.com/langningchen/XMOJ-Script/assets/85378277/1aa93709-30fa-4bd6-b2cd-5d31f2e4f6c9)
